From c22c556a37774a59d7df0217625536c29bf6d386 Mon Sep 17 00:00:00 2001
From: Myron Stowe <myron.stowe@redhat.com>
Date: Sun, 10 Aug 2014 18:20:06 -0400
Subject: [Documentation] MAINTAINERS: add overall IOMMU section

Message-id: <20140810182006.24055.93460.stgit@gir.stowe>
Patchwork-id: 87577
O-Subject: [RHEL7.1 PATCH 06/19] MAINTAINERS: add overall IOMMU section
Bugzilla: 1127467
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Don Dutile <ddutile@redhat.com>
RH-Acked-by: Alex Williamson <alex.williamson@redhat.com>
RH-Acked-by: Nikolay Aleksandrov <nikolay@redhat.com>
RH-Acked-by: Stefan Assmann <sassmann@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1127467
Upstream Status: 0b6e8569b7b767f9418f1b043aa5986015a33b21

commit 0b6e8569b7b767f9418f1b043aa5986015a33b21
Author: Stephen Warren <swarren@nvidia.com>
Date:   Fri Sep 13 13:10:27 2013 -0600

    MAINTAINERS: add overall IOMMU section

    I believe that Joerg Roedel is at least the path through which
    drivers/iommu changes should be merged. Add a MAINTAINERS entry to
    make this clear, so that he's Cd'd on all relevant patches. This is
    relevant for non-AMD/Intel IOMMUs, where get_maintainers.pl doesn't
    currently remind anyone to Cc Joerg on patches.

    Signed-off-by: Stephen Warren <swarren@nvidia.com>
    Signed-off-by: Joerg Roedel <joro@8bytes.org>

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/MAINTAINERS b/MAINTAINERS
index 59c6a2e..0f2c923 100644
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -4345,6 +4345,13 @@ L: linux-serial@vger.kernel.org
 S: Maintained
 F: drivers/tty/serial/ioc3_serial.c
 
+IOMMU DRIVERS
+M: Joerg Roedel <joro@8bytes.org>
+L: iommu@lists.linux-foundation.org
+T: git git://git.kernel.org/pub/scm/linux/kernel/git/joro/iommu.git
+S: Maintained
+F: drivers/iommu/
+
 IP MASQUERADING
 M: Juanjo Ciarlante <jjciarla@raiz.uncu.edu.ar>
 S: Maintained
-- 
1.7.1